application-ckeditor-plugins/src/main/resources/xwiki-upload/plugin.js

```diff
@@ -26,6 +26,8 @@
 
     init: function(editor) {
       preventParallelUploads(editor);
+      listenToUploadedAttachments(editor);
+      listenUploadRequest(editor);
```

Comment:
Why do you need two functions? :) for me they are related. I'd keep a single ``listenToUploadedAttachments``.

Reply:
Ok I'll group them.